master/internal/checkpoints/postgres_checkpoints_intg_test.go

```diff
@@ -378,75 +378,21 @@ func TestDeleteCheckpoints(t *testing.T) {
 	ckpt1 := uuid.New()
 	checkpoint1 := db.MockModelCheckpoint(ckpt1, allocation)
 	err := db.AddCheckpointMetadata(ctx, &checkpoint1, tr.ID)
```

Comment:
Probably need to explicitly ignore this to satisfy linters
```suggestion
	_ = db.AddCheckpointMetadata(ctx, &checkpoint1, tr.ID)
```

Reply:
I think I needed `require.NoError(t, err)`. Lint complains when you don't error check